store-implementation/federated-store/src/test/java/uk/gov/gchq/gaffer/federatedstore/FederatedStoreViewAggregationTest.java

```diff
@@ -132,6 +135,37 @@ public void shouldOnlyReturn1EntitySmallerThanViewFilter() throws Exception {
                 .hasSize(1);
     }
 
+    @Test
+    public void shouldOnlyReturn1EntitySmallerThanSchemaValidationLimit() throws Exception {
```

Comment:
I think this test doesn't make sense in the FederatedStoreViewAggregationTest, maybe should be moved or the test renamed.

Reply:
Ideally, there would be a whole suite of tests that check how different things are merged when schemas are slightly different. These exists elsewhere, but anything to do with ApplyViewToElementsFunction should probably be in one place.

Reply:
"doesn't make sense in the FederatedStoreViewAggregationTest"

Can you explain?

Reply:
The test is called FederatedStoreViewAggregationTest but shouldOnlyReturn1EntitySmallerThanSchemaValidationLimit tests that the ApplyViewToElementsFunction applies schema validation, which has nothing to do with FederatedStoreViewAggregation.

Reply:
> Ideally, there would be a whole suite of tests that check how different things are merged when schemas are slightly different. These exists elsewhere, but anything to do with ApplyViewToElementsFunction should probably be in one place.


asking for tests is always valid for all PR 😆 
You already wrote the tests that addGraphs and getSchema for slight different schema test? a ticket can be opened to pop some elements in those test? or modify them or expand them. `FederatedStoreSchemaTest.java`

Reply:
yes I agree 3058 should resolve that.